src/main/java/net/vivialconnect/model/message/Message.java

```diff
@@ -136,6 +144,12 @@
      */
     private List<String> mediaUrls;
 
+    /**
+     * ID of the bulk to which this message belongs.
+     */
+    @JsonProperty("bulk_id")
+    private String bulkId;
```

Comment:
I'd prefer the Bulk API to be a separate object than re-using the Message class. Should be easier to understand. Thoughts?